packages/components/src/slot-fill/fill.ts

```diff
@@ -29,7 +29,7 @@ export default function Fill( { name, children }: FillComponentProps ) {
 	useLayoutEffect( () => {
 		ref.current.children = children;
 		if ( slot ) {
-			slot.forceUpdate();
+			slot.rerender();
```

Comment:
`forceUpdate` was previously a method of the component class, now `slot` is the `Rerenderable` instance.